cli/js/console_test.ts

```diff
@@ -14,8 +14,8 @@ const customInspect = Deno.symbols.customInspect;
 const {
   Console,
   stringifyArgs
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-} = Deno[Deno.symbols.internal] as any;
+  // @ts-ignore
```

Comment:
Was there another reason to change this?  We should keep the `any`.

Reply:
The issue here is that with the `noImplicitAny` setting set to true (implicitly turned on by the "strict" flag), TypeScript won't allow indexing `Deno` with `Deno.symbols.internal` without adding that symbol as an index in `Deno`'s type declaration.

Another issue there is that I'm not sure that it's possible to add a computed property to a namespace. The only way I could think of to make this work would be to change `Deno` into a `const` like in [this example](http://www.typescriptlang.org/play/index.html?ssl=1&ssc=1&pln=13&pc=37#code/PTAEEEBNIZ1BLAdpeAnApgYwC7wPaKjZ6gCGAbnvJKJmpgK4A2pqoGAZuhopugoQBiePAChIWFhlCJSAW3QwADqT6gA+jACecgEZ4mcAN6jQZ0OgAeSvKmy0CMe7tYAuUA0TwAjg37a9AwBuUQBfUXFJVn5MR3thPHcTc3Z0UkgCJi1QAP1Dd2wtJXQ8Dg1cgxgQlIBtTR08mAA6F1QAXXdEBj1uEPDIzCl+Dk8cfEJVPiVsGAA5bt1uAApEToXuAEp3SmoQ0Un0abn11CWEmoSmisMW1jaNoKA). (Note this example will only type check but won't run.)

Do you think it's worth updating the `Deno` type declaration as described? I'm a bit conflicted because I think it makes sense for `Deno` to be a namespace and I suppose this internal symbol wouldn't be used by user code. What do you think?